core/src/toga/handlers.py

```diff
@@ -50,6 +50,7 @@ async def long_running_task(
         "Use of generators for async handlers has been deprecated; convert "
         "the handler to an async co-routine that uses `asyncio.sleep()`.",
         DeprecationWarning,
+        stacklevel=2,
```

Comment:
This is a pre-existing problem, but brackets around the multi-line string would be preferable.